nncf/openvino/graph/transformations/commands.py

```diff
@@ -64,7 +64,7 @@ def __init__(
         inplace_op_fn: InplaceInsertionFnType,
         fn_output_port_id: int,
         last_inplace_node_name: str,
-        output_dtype: ov.Type = ov.Type.f32,
+        output_dtype: Optional[ov.Type] = None,
```

Comment:
@KodiaqQ I see you've introduced this parameter, is it ok if I change this? I need to collect shapes and they don't have to be in f32.

Reply:
I don't mind.